source/patterns/@aws-solutions-constructs/core/lib/s3-oac-origin.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+
+/*
+ *  The functions found here in the core library are for internal use and can be changed
+ *  or removed outside of a major release. We recommend against calling them directly from client code.
+ */
+
+import { Construct } from 'constructs';
+import * as cloudfront from 'aws-cdk-lib/aws-cloudfront';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+
+/**
+ * Properties to use to customize an S3 Origin.
+ */
+export interface S3OacOriginProps extends cloudfront.OriginProps {
+  /**
+   * An optional Origin Access Identity of the origin identity cloudfront will use when calling your s3 bucket.
+   *
+   * @default - An Origin Access Identity will be created.
+   */
+  readonly originAccessControl: cloudfront.CfnOriginAccessControl;
```

Comment:
The comments imply this property is optional, but the declaration  is missing the `?`. This is probably by design, but looks a bit different than normal. Can you help clarify?

Reply:
Good catch, updated the comments in 721725bcdb9c86e80a149beb36078c136d6f52c6